examples/analytics/01_industry_insights.js

```diff
@@ -0,0 +1,15 @@
+const Bitmovin = require('bitmovin-javascript').default;
+console.log(Bitmovin);
+
+const BITMOVIN_API_KEY = '<YOUR_API_KEY>';
+const bitmovin = new Bitmovin({apiKey: BITMOVIN_API_KEY, debug: false});
+bitmovin.analytics.insights.industry.builder
+  .metric('error_percentage')
+  .industry('all')
+  .subIndustry('test')
+  .filter('browser', 'Firefox')
+  .filter('country', 'Austria')
```

Comment:
We should throw an error or something if two filters are used - otherwise it gets totally undiscoverable to the user